monitor/flowgen/bridging/bridging.go

```diff
@@ -61,7 +67,15 @@ func openOrder(
 		return solvernet.OrderID{}, false, errors.New("dst token not found")
 	}
 
-	expense := app.TokenAmt{Token: dstToken, Amount: conf.orderSize}
+	orderSize, err := estimateOrderSize(ctx, networkID, backend.Client, owner, conf)
+	if err != nil {
+		return solvernet.OrderID{}, false, errors.Wrap(err, "estimate order size")
+	}
+	if orderSize.Cmp(bi.Zero()) == 0 {
```

Comment:
no need to use `Cmp` anymore. See `bi` package, you can use `bi.IsZero` in this case.